File: DuDoong-Api/src/main/java/band/gosrock/api/statistic/query/result/OrderStatistic.java
package band.gosrock.api.statistic.query.result;


import band.gosrock.domain.common.vo.Money;
import java.math.BigDecimal;
import lombok.Getter;

@Getter
public class OrderStatistic {

    private final Long notApprovedCount;
    private final Long DoneCount;
    private final Money sellAmount;

    public OrderStatistic(Long DoneCount, Long notApprovedCount, BigDecimal sellAmount) {
        this.notApprovedCount = notApprovedCount;
        this.DoneCount = DoneCount;
        this.sellAmount = Money.wons(sellAmount.longValue());
    }
}

File: DuDoong-Api/src/main/kotlin/band/gosrock/api/statistic/query/result/OrderStatistic.kt
package band.gosrock.api.statistic.query.result

import band.gosrock.domain.common.vo.Money
import java.math.BigDecimal

class OrderStatistic(doneCount: Long, notApprovedCount: Long, sellAmount: BigDecimal) {
    val DoneCount: Long = doneCount
    val notApprovedCount: Long = notApprovedCount
    val sellAmount: Money = Money.wons(sellAmount.toLong())
}
